src/Badge/Model/Badge.php

```diff
@@ -36,11 +35,11 @@ public function __construct(string $subject, string $status, string $color, stri
     }
 
     /**
-     * @return string the Hexadecimal #FFFFFF.
+     * @return string the Hexadecimal #FFFFFF
      */
     public function getHexColor(): string
     {
-        return '#' . $this->color;
+        return '#'.$this->color;
```

Comment:
we had not said to use the rule:        ` 'concat_space' => ['spacing' => 'one'],`